PKGBUILD

```diff
@@ -53,10 +53,9 @@ declare -gA _system_libs=(
   [harfbuzz-ng]=harfbuzz
   [icu]=icu
   [libdrm]=
-  [libevent]=libevent
```

Comment:
Does removing this system dependency have any benefits?

Reply:
We based the PKGBUILD on the inox one, which was some time ago, libevent is no longer included in the third_party folder.

Reply:
On a side note - the libevent library is at `base/third_party/libevent`. With the removal of Debian's system patch, this dependency isn't needed anymore.